modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -946,22 +950,22 @@ private ObjectEntry _toObjectEntry(
 	}
 
 	private Map<String, Serializable> _toObjectValues(
-			long groupId, ObjectDefinition objectDefinition, long objectEntryId,
-			Map<String, Object> properties, Locale locale)
+			long groupId, Locale locale, ObjectDefinition objectDefinition,
```

Comment:
Before: Locale was the last parameter 
After: locale is before object definition 
Don't we need to sort these parameters? :thinking:

Reply:
I have no idea.. I chose to put it in alphabetical order. Should I change to:

```
long groupId, long userId, ObjectDefinition objectDefinition,
long objectEntryId, Map<String, Object> properties, Locale locale)
```

Reply:
Since we are just adding the `userId`, we could "respect" the actual order and just add the userId as you suggested.
I think the actual order is strange, but it was merged like this so.... :/

Reply:
Done!